### DISCUSSION TITLE
Issues found with jtxBoard within Windows Subsystem for Android

### DISCUSSION THREAD

**Discussion by SomeMichael** [TechbeeAT/jtxBoard#411] [Q&A]
# First to mention
I really like your app!!! :)

# Windows support?
The bad thing is/was: I could not access my data from my windows computer. To my knowledge there is no (windows based) app capable of that.

# Solution
I installed the [Windows subsystem for Android](https://learn.microsoft.com/de-de/windows/android/wsa/) and [sideloaded](https://www.xda-developers.com/how-to-sideload-android-apps-on-windows-11/) F-Droid ([apk](https://f-droid.org/F-Droid.apk)). Then, through F-Droid, installed DAVx5 and jtxBoard and setup all the permissions. All fine.👍

# Issues
Just after a few tests I found two strange behaviors.
1. Notes with (hidden) hyperlinks
When reading a note containing hyperlinks, the links are not visible
![grafik](https://user-images.githubusercontent.com/43578183/227787667-2cee6df0-6a9d-4800-b01d-6dcea0ac6634.png)
In editing mode all shows up as it should
![grafik](https://user-images.githubusercontent.com/43578183/227787711-5e7185e6-2c48-4977-b610-7758c9879134.png)

2. One note is missing in notes overview
One note was not showing up in the notes overview pane at all. It was also not found by searching for it. This note was a linked note from within a task!
![grafik](https://user-images.githubusercontent.com/43578183/227787731-ac269a75-9be9-4140-8c2c-9c626b6cd5af.png)
Notes overview not showing note _TestNote_:
![grafik](https://user-images.githubusercontent.com/43578183/227787740-135714b9-9869-44f0-b59c-da71f9b567b2.png)
Within "native" Android on my smartphone the note shows up as expected after syncing...

These are not big issues from my point of view and there is probably hardly anybody using the Windows Subsystem approach at all, but i wanted to share my approach and my findings here. ;-) Maybe there is an easy solution to it?

Greetings, Michael

**Comment by patrickunterwegs** [comment]
Hey Michael! 

Thank you for your feedback and remarks! :)

About the missing entry:
The default setting hides all sub-entries in the list. But there is a setting in the top menu where you can enabke the flat view which shows you all sub-entries within the list!

About the link issue:
This was already addressed and resolved. Regrettably f-Droid seems to have some issues at the moment and they are not building the latest versions for more than 3 weeks now. 
v2.3.0 is ready and should contain the fix. I hope that those issues there will be solved soon!

**Comment by SomeMichael** [comment]
Good evening Patrick!

> About the missing entry:
> The default setting hides all sub-entries in the list. But there is a setting in the top menu where you can enabke the flat view which shows you all sub-entries within the list!

Oh, ok! I was't aware that this also applies to linked notes.

> About the link issue: F-Droid

Also ok. I can wait... :)

Thanks for your fast response!

**Reply by patrickunterwegs** [reply]
Thanks for the update :-)
I will close this discussion now, if you have any other remarks or issues, please feel free to open another one 😉
